packages/build-utils/package.json

```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "build": "node build",
     "test": "jest --env node --verbose --runInBand --bail",
-    "test-unit": "yarn test test/unit.*test.*",
-    "test-integration-once": "yarn test test/integration.test.ts"
+    "test-unit": "turbo build --filter=@vercel/build-utils && pnpm test test/unit.*test.*",
+    "test-integration-once": "pnpm test test/integration.test.ts"
```

Comment:
This is wrong. It should be only running the integration tests, not the unit tests

Reply:
Oh you're right. I misread that as `pnpm test-unit`. Ignore this comment 🙈